### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by danielkocot** [apideck-libraries/portman#452] [Ideas]
Hi,

I didn't know where to put it right. Is there a roadmap for Portman available? We are using Portman a lot with customers. So would be good to know what the future will bring.

Best,
Daniel

**Comment by thim81** [comment]
hi @danielkocot 

At this moment, we don't have a defined roadmap or fixed features in mind.
Concrete, we are gradually extending support for OpenAPI 3.1 (for which we have taken the 1st steps).
Next to that, we have ideas to build create a GUI to easily configure and manage Portman configurations and a kind of extension that monitors Postman collections with a reporting dashboard.

We are always looking for input from the community, users, and use cases we experience.
I have you requests or ideas, feel free to share them so we can refine & groom them and see what is possible.
